### PR TITLE
Updated CHANGELOG and package.json for 3.2.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-release-3.2.0](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-release-3.2.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.2.0 (2020-04-02)
+
+- Added support for displaying icons from the SKY UX icon font to the icon component. [#74](https://github.com/blackbaud/skyux-indicators/pull/74)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: The back-to-back-to-back prepositional phrases make this a little hard to parse on the first read. Can we switch from `Added` to `Updated` to get rid of one of the prepositional phrases? I'm thinking:

```Updated the icon component to support displaying icons from the SKY UX icon font.`

Or does that change the meaning in some way I'm overlooking?

Comment (Blackbaud-PaulCrowder) [reply CHANGELOG.md:3]: That change works. Thanks!

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-release-3.2.0](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-release-3.2.0)
